### DISCUSSION TITLE
Add ability to teleport to a specific island warp via command

### DISCUSSION THREAD

**Discussion by Skizzles** [BG-Software-LLC/SuperiorSkyblock2#277] [Approved Feature Requests]
Example:

/is warp [Player name] [Warp name]

**Comment by ItchyDitchy** [comment]
Making a fork :)

**Reply by ItchyDitchy** [reply]
Just made a pull request to add the feature...

**Comment by ItchyDitchy** [comment]
Hey there Skizzles, I just got a reply for the fork, what if they type /is warp [warp}, it sounds like a valid command?
